interactions/api/models/member.py

```diff
@@ -83,7 +83,7 @@ def __init__(self, **kwargs):
             self.avatar = self.user.avatar
 
     def __repr__(self) -> str:
-        return self.user.username if self.user else self.nick
+        return self.nick or (self.user.username if self.user else None)
```

Comment:
Can't you just do `return self.name` here?

Reply:
Does that exist?

Reply:
> Does that exist?

You yourself implemented it in this PR.

Reply:
OH GOD I'M STUPID

Reply:
To clarify even more, this line is the same as your `name` property that you implemented, so why not just do:
```py
def __repr__(self) -> str:
    return self.name
```

Reply:
Yeah I got it